addons/web/static/src/js/views/form_upgrade_widgets.js

```diff
@@ -23,7 +23,7 @@ var AbstractFieldUpgrade = {
         this._super.apply(this, arguments);
         this.get_enterprise_label().after($("<span>", {
             text: "Enterprise",
-            'class': "label label-primary oe_inline"
+            'class': "label label-primary oe_inline oe_enterprise_label"
```

Comment:
@tivisse ,
yes there is no use as  css for settings are reviewed that portion to set enterprise label on right side of box was removed. ( i already removed that but may be after merged due to conflict that was here.)